styles/globals.scss

```diff
@@ -10,8 +10,9 @@ body {
   color: #333333;
   line-height: 22.4px;
 
-  b, strong {
-    font-family: 'Source Sans Pro Bold', arial;
+  b,
+  strong {
+    font-weight: 700;
```

Comment:
Note: This is the only substantive change to this file. The rest are auto-linting changes.